examples/api/analysis_options.yaml

```diff
@@ -1,6 +1,11 @@
 # This file is also used by dev/bots/analyze_snippet_code.dart to analyze code snippets (`{@tool snippet}` sections).
 
-include: ../../analysis_options.yaml #TODO(Piinks): Replace with flutter_lints after pub roll.
+# include: ../../analysis_options.yaml #TODO(Piinks): Replace with flutter_lints - separate change.
```

Comment:
@loic-sharma FYI I put this here for now to exclude examples/api from analysis here for now.